### ISSUE TITLE
Automatic Login

### ISSUE THREAD

**Issue by arianofaoms** [thingsboard/thingsboard#3143]
**Component**

Generic

**Description**
Hi there, 
We are in the process of integrating with ThingsBoard and would like to have your insight on seamless login to the ThingsBoard servers we deployed through Azure Cloud. We wish to "log the user in" Thingsboard as soon as our user logs into our web/mobile solution, so that they won't have to login **again** if they choose to navigate/get redirected to their ThingsBoard dashboard directly from our platform.

Thanks for reading this. Looking forward to your response!

Best,

**Environment**

 * OS:  Ubuntu
 * ThingsBoard: PE latest
 * Browser: Chrome/Firefox/Safari::latest

**Comment by vcmorini** [comment]
Hi, I'm also interested in redirecting a user from my app to his/her dashboard in thingsboard.

**Comment by eddyson1006** [comment]
Also interested in any possible approach for automatic login in ThingsBoard from custom platform.

**Comment by patrickwilliamson1** [comment]
Any update on this?

**Comment by X1folta** [comment]
yeah even just a remember me would be a step up. Biometric login would be topnotch for any of our users.

**Comment by cfradewavecom** [comment]
+1

**Comment by mykhailokornieiev** [comment]
Hi. Thank you for the suggestion. Such a feature request was previously added to our backlog for developers. 
We will review this request and will get back to you with updates as soon as they arise. 
Please follow our [Release Notes](https://thingsboard.io/docs/reference/releases/) to see all new additions and fixes to the platform in subsequent releases.

**Comment by mykhailokornieiev** [comment]
Hi. We've created an Idea discussion https://github.com/thingsboard/thingsboard/discussions/15700 so that we can check the community's opinion regarding this request. This is done to identify the most desirable features for the community with further analysis of this data and its transfer to developers for further review. Even if the feature request is rejected, this information will still be useful for further planning the platform's development, and the decision may be reconsidered in the future.
